drfutils/serializers.py

```diff
@@ -68,3 +87,13 @@ def __init__(self, *args, **kwargs):
                     self.fields[field_name] = serializer_class(
                         context=self.context
                     )
+
+
+def SerializerBuilder(serializer_class, fields):
+    serializer_class = type(
+        'CustomSerializer',
```

Comment:
Perhaps this might be nicer to be `serializer_class + 'Custom'` (or `+ `Limiter`)